assets/course-theme/blocks/template-style/template-style.block.json

```diff
@@ -0,0 +1,13 @@
+{
+	"name": "sensei-lms/template-style",
+	"category": "theme",
+	"parent": [],
```

Comment:
Prevents the block from appearing in the block inserter.

Reply:
> I think this block can be a pretty powerful tool that we'll want to provide for the users, so they can add their own CSS.

Sure. Do you want it to be available in the block inserter then?

>  But we'll need to add a way to edit the code. I started a new card for it in the project and will cook up some designs.

Well there is already a way to edit the code. Just do "Edit as HTML" and it allows you to edit the content very easily. It does not provide any formatting though.


https://user-images.githubusercontent.com/2578542/190633369-aa2975b8-c411-492a-8314-cb6c7ecada88.mp4

Reply:
Oh right that works too :)
It's fine that it's not in the inserter yet. Though one thing we can do is lock the block with the lock attribute (`"lock":{"move":true,"remove":true}`) so it's not deleted accidentally.

Reply:
Good idea. Set the `lock` attribute. 663e08acb3dbd8cf3ea02be108226ed20f970fe0